### PR TITLE
[GLUTEN-7887][VL][DOC] Add usage doc about dynamic load jvm libhdfs and native libhdfs3

### PR DESCRIPTION
**Pull request by JkSelf**
## What changes were proposed in this pull request?

Usage doc about how to enable libhdfs and libhdfs3 in runtime.

## How was this patch tested?

No need.

### CODE REVIEW

File: docs/get-started/Velox.md [hunk 1]
```diff
@@ -120,7 +120,8 @@ shared libraries into another jar named `gluten-thirdparty-lib-$LINUX_OS-$VERSIO
 
 ## HDFS support
 
-Hadoop hdfs support is ready via the [libhdfs3](https://github.com/apache/hawq/tree/master/depends/libhdfs3) library. The libhdfs3 provides native API for Hadoop I/O without the drawbacks of JNI. It also provides advanced authentication like Kerberos based. Please note this library has several dependencies which may require extra installations on Driver and Worker node.
+Gluten supports dynamically loading both libhdfs.so and libhdfs3.so at runtime by using dlopen, allowing the JVM to load the appropriate shared library file as needed. This means you do not need to set the library path during the compilation phase.
+To enable this functionality, you must set the JAVA_HOME and HADOOP_HOME environment variables. Gluten will then locate and load the ${HADOOP_HOME}/lib/native/libhdfs.so file at runtime. If you prefer to use libhdfs3.so instead, simply replace the ${HADOOP_HOME}/lib/native/libhdfs.so file with libhdfs3.so.
```
Comment (Yohahaha) [inline docs/get-started/Velox.md:124]: If I replace libhdfs.so with libhdfs3.so, do I still need set `export CLASSPATH=`$HADOOP_HOME/bin/hdfs classpath --glob``?

Comment (JkSelf) [reply docs/get-started/Velox.md:124]: @Yohahaha No need.

**Review by Yohahaha — APPROVED** [review]
👍

File: docs/developers/MicroBenchmarks.md [hunk 2]
```diff
@@ -358,6 +358,11 @@ ShuffleWriteRead/iterations:1/process_time/real_time/threads:1 121637629714 ns
 Unless `spark.gluten.sql.debug` is set in the INI file via `--conf`, the logging behavior is same as debug mode off.
 Developers can use `--debug-mode` command line flag to turn on debug mode when needed, and set verbosity/severity level via command line flags `--v` and `--minloglevel`. Note that constructing and deconstructing log strings can be very time-consuming, which may cause benchmark times to be inaccurate.
 
+
+## Enable HDFS support
+
+After enabling the dynamic loading of libhdfs.so at runtime to support HDFS, if you run the benchmark with an HDFS file, you need to set the classpath for Hadoop. You can do this by running `export CLASSPATH=`$HADOOP_HOME/bin/hdfs classpath --glob``. Otherwise, the HDFS connection will fail.
```
Comment (marin-ma) [inline docs/developers/MicroBenchmarks.md:364]: Is there any way we can use libhdfs3.so for microbenchmark?

Comment (JkSelf) [reply docs/developers/MicroBenchmarks.md:364]: @marin-ma Just use the libhdfs3.so to replace the libhdfs.so in `${HADOOP_HOME}/lib/native`.

Comment (marin-ma) [reply docs/developers/MicroBenchmarks.md:364]: Could you also explain that in the doc? For example, add something like
```
If you have replaced ${HADOOP_HOME}/lib/native/libhdfs.so with libhdfs3.so, there is no need to set the `CLASSPATH`.
```

Comment (JkSelf) [reply docs/developers/MicroBenchmarks.md]: @marin-ma Yes. Added.

**Pull request merged**